backend/src/main/resources/db/migration/V1.9__update_sequence_entries_view_add_pipeline_version.sql

```diff
@@ -0,0 +1,41 @@
+drop view if exists sequence_entries_view;
+
+create view sequence_entries_view as
+select
+    se.*,
+    sepd.started_processing_at,
+    sepd.finished_processing_at,
+    sepd.processed_data as processed_data,
+    sepd.processed_data || em.joint_metadata as joint_metadata,
+    case
+        when se.is_revocation then (select version from current_processing_pipeline)
+        else sepd.pipeline_version
+    end as pipeline_version,
```

Comment:
These lines are the ones I actually changes, just FYI

Reply:
But why not just this?

```suggestion
    (select version from current_processing_pipeline) as pipeline_version,
```

Reply:
We know that this is always the same as `sepd.pipeline_version` as we explicitly filter for it below

```sql
        and sepd.pipeline_version = (select version from current_processing_pipeline)
```

Reply:
Thanks Felix for explaining over a call: the results are different for unprocessed entries and I agree that the current solution is better.